tools/WrapExecutableLinker.sh

```diff
@@ -41,11 +41,12 @@ if [ -f @CMAKE_BINARY_DIR@/tmp/Formaline.sh ]; then
     . @CMAKE_BINARY_DIR@/tmp/Formaline.sh $(basename "${!oindex}")
     temp_files+=("${formaline_output}" "${formaline_object_output}")
     "$@" -DGIT_DESCRIPTION=$git_description -DGIT_BRANCH=$git_branch \
-         ${InfoAtLink_flags} "${InfoAtLink_file}" "${formaline_output}" \
-         ${formaline_object_output}
+         -DLINK_TIME="$(LC_ALL=C date -u)" ${InfoAtLink_flags} \
+         "${InfoAtLink_file}" "${formaline_output}" ${formaline_object_output}
 else
     "$@" -DGIT_DESCRIPTION=$git_description -DGIT_BRANCH=$git_branch \
-         ${InfoAtLink_flags} "${InfoAtLink_file}"
+         -DLINK_TIME="$(LC_ALL=C date -u)" ${InfoAtLink_flags} \
+         "${InfoAtLink_file}"
```

Comment:
Can we use ISO 8601 for simpler parsing? (Then the locale won't matter). `date -Iseconds` works with both the gnu and BSD (e.g. apple) `date` command.

Reply:
Sure.  Changed.